test/CovidTimeseriesModel.py

```diff
@@ -47,21 +48,79 @@ def test_calculate_r(self):
             15/10
         )
 
-    def test_cds_dataset(self):
-        self._test_model(CDSDataset(), self.interventions, 4, 'cds_data')
+    def test_model(self):
+        """Compare the output of the model in it's current state to the outputs it's had in the past"""
+        self._compare_snapshot_tests(
+            self._build_test_parameter_sets(),
+            self._get_snapshots()
+        )
+
+
+    def _build_test_parameter_sets(self):
+        dataset = CDSDataset()
+        states = dataset.get_all_states_by_country('USA')
+        MODEL_INTERVAL = 4
+        sets = []
+        for state in states[:2]:
+            for intervention in self.interventions:
+                sets.append(
+                    CovidUtil().initialize_model_parameters({
```

Comment:
These would need to match up with whatever is in run.py for the pipeline and I can imagine them getting out of sync over time. For now probably fine, but I anticipate that this will needed to be updated - I also know there was talk about having different assumptions for each location, which could change dramatically based on location.

Reply:
Those files were from the old snapshot framework, which depended on them. I removed them because the new framework uses a different format (with a single file). I would argue that removing them simultaneous to the snapshot testing refactor is appropriate.